### DISCUSSION TITLE
Confused about building latest stable version

### DISCUSSION THREAD

**Discussion by tdrz** [baresip/baresip#2002] [Q&A] [answered]
Sorry if this is covered somewhere else, but I can't seem to find it.

Let's say I want to build and run from source the latest stable version of baresip. I guess this is [v2.5.1](https://github.com/baresip/baresip/tree/v2.5.1) as seen in the tags. Since libre and librem are needed (and I also want to build those from source), these are the steps that I am following:

```
$ git clone https://github.com/baresip/re && cd re && git checkout tags/v2.5.0 -b v2.5.0 && make RELEASE=1 && sudo make install && sudo ldconfig
$ git clone https://github.com/baresip/rem && cd rem && git checkout tags/v2.5.0 -b v2.5.0 && make && sudo make install && sudo ldconfig
$ git clone https://github.com/baresip/baresip && cd baresip && git checkout tags/v2.5.1 -b v2.5.1 && make RELEASE=1
$ ./baresip
./baresip: symbol lookup error: ./baresip: undefined symbol: mtx_alloc
```

So this fails. What am I doing wrong?

**Comment by sreimers** [comment, marked as answer]
Are multiple libre.so versions installed?

```sh
$ ls -lha /usr/lib/libre.so*
```

**Reply by tdrz** [reply]
Thank you! That was the issue!
